totalRP3/Modules/TargetFrame/TargetFrame.lua

```diff
@@ -133,7 +133,7 @@ local function onStart()
 				uiAlert:Show();
 				uiAlert:SetWidth(buttonSize / 2);
 				uiAlert:SetHeight(buttonSize / 2);
-				uiAlert:SetTexture(buttonStructure.alertIcon);
+				uiAlert:SetAtlas(buttonStructure.alertIcon);
```

Comment:
I guess our only uses of alertIcon are for atlases now, but it'd probably be worth supporting them both anyway.

```suggestion
				if C_Texture.GetAtlasInfo(buttonStructure.alertIcon) then
					local useAtlasSize = false;
					uiAlert:SetAtlas(buttonStructure.alertIcon, useAtlasSize);
				else
					uiAlert:SetTexture(buttonStructure.alertIcon);
				end
```